dockerfiles/centos/is/init.sh

```diff
@@ -17,7 +17,7 @@
 set -e
```

Comment:
@chamindi-a AFAIU you have added the incorrect Docker container entry point script to WSO2 Identity Server Docker image resources. Please correct this.